Objects/lnotab_notes.txt

```diff
@@ -1,7 +1,7 @@
 Description of the internal format of the line number table in Python 3.10
 and earlier.
 
-(For 3.11 onwards, see Objects/locations.md)
+(For 3.11 onwards, see InternalDocs/locations.md)
```

Comment:
Now that `locations.md` has been removed, it looks incorrect. Perhaps it should be pointed to `InternalDocs/code_objects.md`?